domain-models-runtime/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.folio</groupId>
     <artifactId>raml-module-builder</artifactId>
-    <version>33.3.0-SNAPSHOT</version>
+    <version>33.4.0-SNAPSHOT</version>
```

Comment:
This is a breaking change requiring a major version bump: 33 -> 34.

Reply:
Done. Could you please review?